dattebayo/src/components/Cards/FeaturedCards/FeaturedCard.tsx

```diff
@@ -0,0 +1,18 @@
+export function FeaturedCard() {
+	return (
+		<div className='featuredCard w-60 h-[23rem] bg-slate-900 shadow-xl rounded-xl'>
```

Comment:
Looks good. If you'd like extract the classNames of the elements i.e. assign the value to a variable. E.g. className={featuredCardClass}

Reply:
I doubt I'm going to reuse that exact styling for the element to style other components, but if I do i'll definitely do that 👍🏽